api/app/signals/apps/api/v1/serializers/signal.py

```diff
@@ -292,6 +292,7 @@ class Meta:
         fields = (
             '_links',
             '_display',
+            'id',
```

Comment:
Do we want to expose the `id` on the public signal endpoint?

Reply:
The person putting in the complaint knows that ID through the "SIA meldingsnummer" anyway. I will remove it now, because adding it later is easier than removing it.

Reply:
True, but this will suggest a real ID the "SIA meldingsnummer" is a bit more "abstract". But you got a point here.

Reply:
Is removed now, adding it later is easier than removing it. The `id` is not secret and is known to the person with the complaint (it is part of the SIA number that is assigned upon creation).

Reply:
will add it again, as our frontend uses it